commands/src/main/java/org/jclouds/karaf/commands/table/internal/ScriptEngineShellTable.java

```diff
@@ -51,8 +52,9 @@ public String evaluate(Object obj, String expression) {
     try {
       scriptEngine.put(getType(), obj);
       result = String.valueOf(scriptEngine.eval(expression));
-    } catch (Exception ex) {
-      //Ignore
+    } catch (Exception exception) {
+       result = format("Unable to evaluate expression %s due to: %s. Please check your shell confugration",
```

Comment:
Hmmmm that text will go in a column and it will probably be trimmed or difficult to read. WDYT about changing it to "## VALUE ERROR ##" or whatever short error marker we like, and move the detailed message to the log, with the complete stacktrace?